packages/larva-css/src/03-utilities/u-border.common.inline.scss

```diff
@@ -55,6 +55,12 @@
 	}
 }
 
+.lrv-u-border-t-3\@desktop {
```

Comment:
This can be added in the breakpoint grouping below

Reply:
@laras126 Done.